info/flow_and_pressure.md

```diff
@@ -0,0 +1,42 @@
+
+## Puck Resistance
+
+The pressure at the puck is often treated as a static variable in a commercial setting because it is the pump pressure that is shown. However, it is easy to see with a group pressure gauge that this is not always the case. The resistance in a puck is determined by the grind distribution and the viscosity of the espresso being brewed. It is harder to push honey through a porous medium than it is water, for example. Thus, the pressure generated will depend on grind setting, burr geometry, and the roast profile and origin/variety of the coffee being brewed. At the beginning of a brew when the coffee is at its most soluble, the TDS of the resulting liquid will be very high and output a high pressure. As the viscous solubles of the coffee are depleted, the resulting liquid will often times not flow fast enough to hold the OPV set pressure and the group-pressure reading will begin to fall. Note that on machines which can provide a enough flow, the shot will instead speed up drastically in order to hold pressure. This can often result astringency due to the formation of channels. 
+
+## Pressure Measurement in Espresso
+
+As noted in the pump machine section, the location of the pressure gauge matters in relation to where it is reading in relation to the major sources of resistance, such as the OPV, gicleur/needle-valve, and the espresso puck. Because Bernoulli's law is an expression of the conservation of energy, flow restriction at any given part in the circuit will result in decreased static pressure readings due to either frictional losses or an increase in fluid kinetic energy. Thus, it is important to know where the pressure gauge is installed on an espresso machine when judging what the brew pressure is. The Lelit Bianca pump gauge will give a much different reading than the grouphead gauge due to the OPV, needle valve, and puck. Even within an espresso puck, pressure is not a static reading. Because the opening at the bottom is at atmospheric pressure, there is a necessary pressure drop from the indicated pressure at the top of the puck, inside of the puck, and at the bottom of the puck.  All of this is to say that brew pressure is a somewhat convoluted variable. A turbo shot that pulls 15:40 in 15 seconds might run at a different pressure with someone who is using a Niche + a vibration pump machine versus someone who is using an EK43 + an unrestricted rotary pump machine, but both can turn out equally delicious.
```

Comment:
Bernoulli's principle is not applicable here because of viscous losses. It only accounts for _reversible_ energy transfer mechanisms like going from pressure to kinetic energy and back. It does not account for _irreversible_ energy losses like viscous/frictional losses. Flow restrictions like a gicleur or needle valve will have significant viscous losses so you can't really use Bernoulli's principle there. You can however use a modified version of Bernoulli's principle that accounds for "head losses" in the system.